doc/classes/EditorInterface.xml

```diff
@@ -285,6 +285,47 @@
 				See also [method Window.set_unparent_when_invisible].
 			</description>
 		</method>
+		<method name="popup_dialog_property_selector">
+			<return type="void" />
+			<param index="0" name="object" type="Object" />
+			<param index="1" name="callback" type="Callable" />
+			<param index="2" name="type_filter" type="PackedInt32Array" default="PackedInt32Array()" />
+			<description>
+				Pops up an editor dialog for selecting properties from [param object]. The [param callback] must take a [NodePath]. It is [method Callable.call_deferred] on the selected property path (see [method NodePath.get_as_property_path]) or the empty path [code]^""[/code] if the dialog is canceled. If [param type_filter] is provided, the dialog will only show properties that match one of the listed [enum Variant.Type] values.
```

Comment:
```suggestion
				Pops up an editor dialog for selecting properties from [param object]. The [param callback] must take a single argument of type [NodePath]. It is [method Callable.call_deferred] on the selected property path (see [method NodePath.get_as_property_path]) or the empty path [code]^""[/code] if the dialog is canceled. If [param type_filter] is provided, the dialog will only show properties that match one of the listed [enum Variant.Type] values.
```
> It is [method Callable.call_deferred] on the selected property path

This sounds super awkward, but not sure how it should be corrected. There is no reason to mention "call_deferred", as the callback is asynchronous anyway (it's called when user takes action).

Reply:
I figured it might be nice for the user to know that the callback will not execute immediately, but rather one frame after the item gets selected.

Reply:
But it does not make any difference, as the action happens on button click. There is no code difference on the user side if the callback is called immediately or later, unless they for whatever reason accessed the dialog node and manually connected to the button (unlikely).